oe_whitelabel.info.yml

```diff
@@ -23,6 +23,7 @@ regions:
   header_top: Header top
   header_left: Header left
   header_right: Header right
+  banner: Banner
```

Comment:
Maybe we could call this in a more generic way, like "Page header" or "Content top" or something similar.

Reply:
You mean use one of the existing regions?
My goal was to put this outside of `class="container"`  elements, to have natural full width.
It is not fully needed, because the element itself has `100vw`, but if we already move it to a separate region, we can go all the way.

In fact the `width: 100vw` is a bit clumsy, because it does not subtract the width of the vertical scrollbar. So it means we need an `overflow-x: hidden` somewhere.
But as long as the 100vw is in bcl, it does not really matter which region we use..

What's worse, all the regions following this one are within `class="row"`, which messes with the container padding.
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-1226?jql=project%20%3D%20oel%20AND%20text%20~%20%22container%20padding%20row%22
In our current solution, this required us to add classes from outside to mitigate the problem.

Reply:
Fyi, the first comment in OEL-1226 has a screenshot that shows a symptom of the 100vw problem. Part of the text for "full-width section for bcl-content-banner" is cut off on the left.

Reply:
A good experiment is to put a block into the sidebar region.
We don't want the banner to show up next to the sidebar.

Reply:
The place where you put the new region is perfect, I didn't like the name much. We had a chat with @drishu and we will follow the naming used in Olivero core theme: `hero: 'Hero (full width)'`